client/components/status/status.js

```
@@ -303,11 +328,30 @@ export default class Status extends React.Component {
       return;
     }
 
+    if (!this.state.activeSessions.length) {
```

ESLint

[error] Must use destructuring state assignment

(react/destructuring-assignment)